src/components/input-company.js

```diff
@@ -1,5 +1,5 @@
 /** @jsx jsx */
-import React from 'react';
```

Comment:
Someone is using Single quotes instead of double or doesn't use Prettier before saving.